guide/src/class.md

```diff
@@ -195,7 +195,7 @@ Python::with_gil(|py|{
 
 ## Customizing the class
 
-{{#include ../../pyo3-macros/docs/pyclass_parameters.md}}
+{{#include ../pyclass_parameters.md}}
 
 [params-1]: {{#PYO3_DOCS_URL}}/pyo3/prelude/struct.PyAny.html
```

Comment:
You added these links to `pyclass_parameters.md` - does that create a problem with the duplicate definitions?

Reply:
I don;t think so, but I've fixed it.

Reply:
I think I slightly preferred the (slightly tedious) duplication of having the links written out in `class.md` and `lib.rs` because that way the links were mostly adjusted to link within the current version of the docs than always be stuck pointing at `latest`, though it's not a huge deal.